dashboard/app/views/layouts/_header.html.haml

```diff
@@ -10,6 +10,8 @@
 
 - show_bug_links = (script_level && script.show_report_bug_link?) || (level && level.try(:is_project_level))
 
+= render partial: 'shared/header_progress_popup'
```

Comment:
I suppose, strictly speaking, this is no longer necessary.  When I was trying for a pure-css solution the original parenting was problematic because the header anchors to a zero-height absolutely-positioned element.  I moved the popup out of this hierarchy so I could anchor it to something body- or viewport-sized and set the height that way.

Now we get the viewport height with `$(window).height()` in script so it shouldn't matter.  Shall I move the markup back to its original location?